main.go

```diff
@@ -4,10 +4,37 @@ import (
 	"fmt"
 )
 
+var unusedVar int
+
+type Hello struct {
+	Name   string
+}
+
 func add(x int, y int) int {
-	return x + y 
+	return x - y 
+}
+
+func exception() {
+	defer func() {
+		r := recover()
+		if r == nil {
+			fmt.Println("Nothing was wrong")
+		} else {
+			fmt.Println("Recovered from", r)
+		}
+	}()
+
+	var p *int
+	*p = 0
```

Comment:
Avoid intentionally causing a panic in the `exception` function. If you want to demonstrate error handling, consider using a more realistic example. [important]